### DISCUSSION TITLE
Page Templates / Calendar Template

### DISCUSSION THREAD

**Discussion by Balky79** [redimp/otterwiki#131] [Ideas]
Hi, 

I understand the whole idea of having a simple wiki, but as usual - when good ideas come to life, some people like to complicate a bit. :D 

I'm using otter wiki to note my daily journal, and I'd love to have simple - but automated way - of having Index page that is like a calendar table. 

In it, links with certain "tag". Now, tags are bonus, but if calendar page would have some setting like - "list here all pages starting with "24cal_", "24w_cal" - that would put links into appropriate boxes (days), that would be awesome. 

Just an idea, I'll not be offended if rejected :) 
Keep up the great product!

**Comment by redimp** [comment]
Hey @Balky79,

thank you for both sharing how you make use of An Otter Wiki and bringing up the discussion about these two features!

tags: I've considered to add the option of adding tags, for example via a yaml front matter or a h5 style headline, something like this will come up eventually.

(configurable/filterable) page index: I see how this can be useful! Making this configurable so that a selection of pages is displayed, where the pages are filtered via wildcard name match, via tags or even via a configured search sounds like a cool feature. And adding this would fill the current gap of being unable to mix page index and page with content.

I will put some thought into this and upon up issues. Will tag you there.
